compiler/il/Aliases.cpp

```diff
@@ -797,6 +797,10 @@ OMR::SymbolReference::setSharedShadowAliases(TR_BitVector * aliases, TR::SymbolR
       while (symRef = i.getNext())
          if (symRef->getSymbol() == self()->getSymbol())
             aliases->set(symRef->getReferenceNumber());
+
+      // include symbol reference's own shared alias bitvector
```

Comment:
The current convention appears to be calling setReallySharesSymbol() to set the flag right after a symbol reference is created, so setting the flag in makeSharedAliases might not make the most sense if a symbol reference wasn't created initially to share the underlying symbol. However, I can add an assert in makeSharedAliases when a symbol reference is passed in without its flag being set to address the case you brought up?